### PR TITLE
Fixed casing according to english version

### PR DESCRIPTION
**Pull request by KonkenBonken**
Before the german casing was much more generous, I changed it to be much more consistent and uniform according to the english version.

### CODE REVIEW

File: Schlange.py [hunk 1]
```diff
@@ -84,42 +84,42 @@ class KeywordTranslator:
     'KeyboardInterrupt': 'Tastaturunterbrechung',
     }
     errors_with_placeholders = {
-    'unsupported operand type': 'nicht unterstützter Operandentyp',
-    'division by zero': 'Division durch Null',
```
Comment (KonkenBonken) [inline Schlange.py:88]: `Null` with an upper-case `N` is likely to be confused with static value keywords, such as `True`, `False` and `None`, but here, it refers to the number `0`

File: Schlange.py [hunk 2]
```diff
@@ -84,42 +84,42 @@ class KeywordTranslator:
...
-    'division or modulo by zero': 'Division oder Modulo durch Null',
-    'key error': 'Schlüsselfehler',
-    'invalid literal for int() with base {}: \'{}\'': 'Ungültiges Literal für int() mit Basis {}: \'{}\'',
-    'unexpected indent': 'Unerwartete Einrückung',
-    'not a valid identifier': 'Kein gültiger Bezeichner',
-    'invalid syntax, unexpected {}: \'{}\'': 'Ungültige Syntax, unerwartetes {}: \'{}\'',
-    'unsupported operand type(s) for {}: \'{}\' and \'{}\'': 'Nicht unterstützte Operandentypen für {}: \'{}\' und \'{}\'',
+    'unsupported operand type': 'nicht unterstützter operandentyp',
+    'division by zero': 'division durch null',
+    'name \'{}\' is not defined': 'name \'{}\' ist nicht definiert',
```
Comment (KonkenBonken) [inline Schlange.py:89]: All of the english errors starts with a lower-case letters, so the german ones should as well

File: Schlange.py [hunk 3]
```diff
@@ -84,42 +84,42 @@ class KeywordTranslator:
     'KeyboardInterrupt': 'Tastaturunterbrechung',
     }
     errors_with_placeholders = {
-    'unsupported operand type': 'nicht unterstützter Operandentyp',
-    'division by zero': 'Division durch Null',
-    'name \'{}\' is not defined': 'Name \'{}\' ist nicht definiert',
-    'list index out of range': 'Listenindex außerhalb des gültigen Bereichs',
```
Comment (KonkenBonken) [inline Schlange.py:90]: And there is no real reason for non-first words to start with an upper-case letter, and the english version has no upper-case

File: test.py
Comment (KonkenBonken) [inline test.py:1]: According to [63373a2](https://github.com/KonkenBonken/Swedish-Python-Interpreter/commit/63373a28cd69e26cc6c829f60bf5791bc5d19c41#diff-d9e78e6195c8416fa498aef198b60384be105893b476190278cf92a596efed14R42)

File: Schlange.py [hunk 4]
```diff
@@ -39,7 +39,7 @@ class KeywordTranslator:
         "selbst" : "self",
         "drucke" : "print",
         "mathe" : "math",
-        "Wurzel" : "sqrt"
+        "wurzel" : "sqrt"
```
Comment (KonkenBonken) [inline Schlange.py:42]: Noticed this was also changed in @AntonPieper's commit: [a89c7fd](https://github.com/actopozipc/German-Python-Interpreter/pull/1/commits/a89c7fdc404d88427862bcbc42e6ee57e660714b#diff-d9e78e6195c8416fa498aef198b60384be105893b476190278cf92a596efed14R42)

**Comment by actopozipc** [comment]
German nouns always start with an upper-case letter, so die Division, der Name and so on.  But maybe Division durch 0 instead of Division durch Null?

**Comment by KonkenBonken** [comment]
Oh, sorry, I don't know German
Hence, the name of my fork repository

**Comment by AntonPieper** [comment]
I thought wurzel would make sense in a naming conventions sense, because in programming languages like python, everything is written in camel case except class names which are written in PascalCase. IMO Naming `sqrt` `Wurzel` would suggest that it is a class

**Pull request closed without merging**